lib/spack/spack/test/ci.py

```diff
@@ -30,7 +30,7 @@ def test_urlencode_string():
     assert ci._url_encode_string("Spack Test Project") == "Spack+Test+Project"
 
 
-@pytest.mark.not_on_windows("Not supported on Windows (yet)")
+@pytest.mark.not_on_windows("gpg supported on Windows")
```

Comment:
```suggestion
@pytest.mark.not_on_windows("gpg not supported on Windows")
```

Reply:
I added the 'not' to the description but this does not show the change.